packages/customWidgets/slider-web/tests/e2e/specs/Slider.spec.ts

```diff
@@ -96,11 +96,12 @@ describe("Slider widget", () => {
             const sliderWidget = new SliderWidget("slider");
 
             const markers = sliderWidget.getMarkers();
-            expect(markers).toHaveLength(10);
+            const markerSize = markers.length;
+            expect(markerSize).toBe(10);
```

Comment:
Isn't the previous revision inline with our convention of using `toHaveLength`? How did this fix e2e tests, does the length change?

Reply:
e2e uses jasmine, which doesnt have this assertion